pai-management/doc/cluster-bootup.md

```diff
@@ -8,32 +8,57 @@ Please refer to Section [single box deployment](./single-box-deployment.md) if u
 ## Table of contents:
 <!-- TOC depthFrom:2 depthTo:3 -->
 
-- [Overview](#overview)
-- [Quick deploy with default settings](#quickdeploy)
-- [Customized deploy](#customizeddeploy)
+- [OpenPAI deployment](#overview)
+  - [Customized deploy](#customizeddeploy)
+  - [Single Box deploy](#singlebox)
+- [Troubleshooting](#problem)
+  - [Troubleshooting OpenPAI services](#troubleshooting_1)
+  - [Troubleshooting Kubernetes Clusters](#troubleshooting_2)
+  - [Getting help](#troubleshooting_3)
+- [OpenPAI Maintenance](#maintenance)
 - [Appendix: Default values in auto-generated configuration files](#appendix)
 
 <!-- /TOC -->
 
-## Overview <a name="overview"></a>
+## OpenPAI deploy <a name="overview"></a>
 
 We assume that the whole cluster has already been configured by the system maintainer to meet the [Prerequisites](../../README.md#how-to-deploy).
 
+
+## Table of contents
+
+- [Customized deploy](#customizeddeploy)
+- [Single Box deploy](#singlebox)
+
 With the cluster being set up, the steps to bring PAI up on it are as follows:
 
-- Step 0. Prepare dev-box
-- Step 1. Prepare PAI configuration.
-    - (For advanced users) This step can either be done by writing the configuration files manually,
-    - (For novice users) or be done using the `paictl` tool.
-- Step 2. Boot up Kubernetes.
-- Step 3. Start all PAI services.
+## Customized deploy <a name="customizeddeploy"></a>
+
+### Steps:
+- [Step 0. Prepare the dev-box](#c-step-0)
+- [Step 1. Prepare the quick-start.yaml file](#c-step-1)
+- [Step 2. Generate OpenPAI configuration files](#c-step-2)
+- [Step 3(Optional). Customize configure OpenPAI](#c-step-3)
+- [Step 4. Boot up Kubernetes](#c-step-4)
+- [Step 5. Start all PAI services](#c-step-5)
+
+### Step 0. Prepare the dev-box <a name="c-step-0"></a>
 
-## Quick deploy with default settings <a name="quickdeploy"></a>
-### Step 0. Prepare the dev-box
 It is recommended to perform the operations below in a dev box.
 Please refer to this [section](./how-to-setup-dev-box.md) for the details of setting up a dev-box.
 
-### Step 1. Prepare the quick-start.yaml file <a name="step-1a"></a>
+#### How to check
```

Comment:
Hi @YanjieGao , we may need to add some instructions about starting a dev-box.

Reply:
will try to add the main instructions here